google/ggrc-core
# GGRC-8272 Create pagination on '_notifications/show_pending' page

# Issue description

HTTP response for getting pending notifications are too large.
Endpoint has been provided in the scope of `GGRC-4549 Fix to show pending notifications` [BE PR](https://github.com/google/ggrc-core/pull/10532)

# Steps to test the changes

1. Log into GGRC as Admin.
2. Go to url `/_notifications/show_pending`.
3. Try to load other notification chunk via pagination component.
4. Expected: Notifications page works.

# Solution description

Added notifications page with pagination for ability to show notifications by chunks.

# Sanity checklist




# PR Review checklist
